### PR TITLE
Feature/adjust gateway

### PR DESCRIPTION
**Pull request by Nazaziik**

### CODE REVIEW

File: Edu-platform_Gateway/ocelot.Production.json [hunk 1]
```diff
@@ -1,207 +1,151 @@
...
       "DownstreamScheme": "https",
       "DownstreamHostAndPorts": [
         {
           "Host": "localhost",
-          "Port": 8070
+          "Port": 8010
         }
       ],
-      "DownstreamPathTemplate": "/api/Analytics"
+      "DownstreamPathTemplate": "/api/Role/{id}",
```
Comment (VitalikMatvieiev) [inline Edu-platform_Gateway/ocelot.Production.json]: @Nazaziik let's use small letter

Comment (VitalikMatvieiev) [reply Edu-platform_Gateway/ocelot.Production.json]: https://learn.microsoft.com/en-us/aspnet/web-api/overview/web-api-routing-and-actions/attribute-routing-in-web-api-2

Comment (VitalikMatvieiev) [reply Edu-platform_Gateway/ocelot.Production.json]: Or did you find different convention rules?

Comment (Nazaziik) [reply Edu-platform_Gateway/ocelot.Production.json]: Fixed

**Review by VitalikMatvieiev — APPROVED** [review]

**Review by tanyaivattt — APPROVED** [review]

**Review by tanyaivattt — APPROVED** [review]

**Review by kossvat — APPROVED** [review]

**Pull request merged**